### PR TITLE
Use Keycloak for JWT creation for AB#14504

### PR DESCRIPTION
**Pull request by sslaws**
Swap out local JWT creation and use Keycloak client credentials grant using Refit.

### CODE REVIEW

**Review by ytqsl — APPROVED** [review]
As a comment for further discussions, https://identitymodel.readthedocs.io/en/latest/ already encapsulates all the required logic to talk to an OAuth2 server

**Review by BrianMaki — APPROVED** [review]

File: Apps/TicketManagement/Services/RedisTicketService.cs [hunk 1]
```diff
@@ -171,43 +174,17 @@ private static string GetRoomName(RoomConfiguration config, string roomType)
             return $"{{{config.Name}}}:Room:{roomType}";
         }
 
-        private (string Token, long Expires) CreateJwt(RoomConfiguration roomConfig, Guid id)
+        private async Task<(string Token, long Expires)> CreateJwt(RoomConfiguration roomConfig)
```
Comment (MikeLyttle) [inline Apps/TicketManagement/Services/RedisTicketService.cs:177]: "Async" should be appended to method name.

File: Apps/TicketManagement/Services/RedisTicketService.cs [hunk 2]
```diff
@@ -252,7 +229,7 @@ private static string GetRoomName(RoomConfiguration config, string roomType)
             return (participantCount, waitingCount, position);
         }
 
-        private void CheckIn(ITransaction transaction, RoomConfiguration roomConfig, Ticket ticket, long? nextCheckIn = null)
+        private async Task CheckIn(ITransaction transaction, RoomConfiguration roomConfig, Ticket ticket, long? nextCheckIn = null)
```
Comment (MikeLyttle) [inline Apps/TicketManagement/Services/RedisTicketService.cs:232]: "Async" should be appended to method name.

File: Apps/TicketManagement/Services/RedisTicketService.cs (hunk 1 above)
Comment (sslaws) [reply Apps/TicketManagement/Services/RedisTicketService.cs]: I've updated all of the methods to include async

**Review by MikeLyttle — APPROVED** [review]

**Pull request merged**